dgraphtest/cluster.go

```diff
@@ -17,10 +17,13 @@
 package dgraphtest
 
 import (
+	"testing"
+
 	"github.com/dgraph-io/dgo/v210"
 )
 
 type Cluster interface {
 	AssignUids(num uint64) error
 	Client() (*dgo.Dgraph, error)
+	SkipTest(t *testing.T, commit string) error
```

Comment:
This function can be common across all clusters. It shouldn't be part of the interface. Instead a funtion `GetVersion` should be that the SkipTest can use

Reply:
Could you please elaborate more on this

Reply:
Think about this, if I have a cloud cluster and also a cluster running on AWS and cluster running on my own machine. Now, in all 3 cases, I would like to skip the test. Would I be implementing the whole SkipTest function again for 3 different clusters?